src/components/shared/__tests__/AddRemoveForm.spec.js

```diff
@@ -72,10 +72,39 @@ describe('<AddRemoveForm />', () => {
 
   it('should show recent items only when recentOnly is true')
 
-  it('should show a message when no matches are found')
+  it('should show a message when no matches are found', async () => {
+    const emptyItemsProps = {
+      ...mockProps.questions,
+      fetchItems: () => Promise.resolve([])
+    }
+
+    wrapper = await shallow(<AddRemoveForm {...emptyItemsProps} />)
+    await wrapper.update()
+
+    let result = wrapper.find({ 'data-test': 'none-available-message' })
+    expect(result.length).toBe(1)
+    expect(result.text()).toBe('No  Matches')
+
+    wrapper.instance().toggleRecentOnly({ preventDefault: jest.fn() })
+    result = wrapper.find({ 'data-test': 'none-available-message' })
+    expect(result.length).toBe(1)
+    expect(result.text()).toBe('No Recent Matches')
+  })
 
   it('should not render buttons for the selectedItems')
 
-  it('should not render HTML in labels buttons')
+  it('should not render HTML in button labels', async () => {
+    const failedProps = {
+      ...mockProps.questions,
+      fetchItems: () => Promise.resolve([{ title: '<script> tags not allowed' }])
+    }
+
+    wrapper = await shallow(<AddRemoveForm {...failedProps} />)
+    await wrapper.update()
+
+    const result = wrapper.find({ 'data-test': 'item-button' })
+    expect(result.length).toBe(1)
+    expect(result.render().html()).toBe('<span>&lt;script&gt; tags not allowed</span>')
+  })
```

Comment:
Trying to do `result.render().text()` actually gave me back the string version of `'<script> tags not allowed'`. Something under the hood must convert `&lt; &gt;` to actual textual values, so we had to use the `.html()` instead.